admin_interface/locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,317 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-12-04 10:23+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: django-admin-interface/admin_interface/admin.py:29
+msgid "Environment"
+msgstr "Umgebung"
+
+#: django-admin-interface/admin_interface/admin.py:41
+msgid "Language chooser"
+msgstr "Sprachauswahl"
+
+#: django-admin-interface/admin_interface/admin.py:52
+msgid "Logo"
+msgstr ""
```

Comment:
You should fill in translations that are identical, so that tools that inspect PO files can report that it’s 100% complete or not 🙂

Reply:
> You should fill in translations that are identical, so that tools that inspect PO files can report that it’s 100% complete or not slightly_smiling_face

Updated.

Double-checked the Django admin catalog and leaves only default message strings blank, as it says in the .po file as a hint.